src/language-css/utils/index.js

```diff
@@ -276,7 +276,8 @@ function isSCSSMapItemNode(path) {
   // Check open parens contain key/value pair (i.e. `(key: value)` and `(key: (value, other-value)`)
   if (
     !isKeyValuePairInParenGroupNode(node) &&
-    !(parentParentNode && isKeyValuePairInParenGroupNode(parentParentNode))
+    !(parentParentNode && isKeyValuePairInParenGroupNode(parentParentNode)) &&
+    !isKeyValuePairNode(node)
```

Comment:
Should we revert this line?

Reply:
Thank you! [ee114cb](https://github.com/prettier/prettier/pull/14089/commits/ee114cbcc708654cf063593356602da8ebf18e19)